src/main/scala/uk/gov/hmrc/play/http/ws/WSRequest.scala

```diff
@@ -24,14 +24,9 @@ import uk.gov.hmrc.http.{HeaderCarrier, Request}
 
 trait WSRequest extends Request {
 
-  import play.api.libs.ws.WS
+  override val configuration: Option[Config]
 
-  override lazy val configuration: Option[Config] = Play.maybeApplication.map(_.configuration.underlying)
-
-  def wsClient: WSClient = {
-    import play.api.Play.current
-    WS.client
-  }
+  val wsClient: WSClient
```

Comment:
This is a breaking change and should not be introduced. Both wsClient and configuration provide a backwards compatible default value and can be overridden if DI is used